inference/v1/models/rfdetr/rfdetr_backbone_pytorch.py

```diff
@@ -336,4 +367,11 @@ def get_dinov2_weight_decay_rate(name, weight_decay_rate=1.0):
         or ("embeddings" in name)
     ):
         weight_decay_rate = 0.0
-    return weight_decay_rate
\ No newline at end of file
+    return weight_decay_rate
+
+
+_current_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
64x spedup sounds nice, but really its moving the loading the files to module load time, which given that this might not get instantiated at all might actually slow down things down on average?